src/interfaces/IOracle.sol

```diff
@@ -5,6 +5,7 @@ pragma solidity >=0.5.0;
 /// @author Morpho Labs
 /// @custom:contact security@morpho.org
 /// @notice Interface that oracles used by Morpho must implement.
+/// @dev The oracle is trusted to handle stale prices and price manipulations if necessary.
```

Comment:
```suggestion
/// @dev The oracle is responsible for handling stale prices and price manipulations if desired.
```

Reply:
we can also harmonize "it is the responsibility of" and "... is responsible for" but it's a tiny detail, I honestly don't care